src/instructlab/sdg/generate_data.py

```diff
@@ -444,11 +444,8 @@ def generate_data(
             empty_sdg_leaf_nodes.append(leaf_node_path)
             logger.warning("Empty dataset for qna node: %s", leaf_node_path)
             continue
-        generated_data = (
-            [new_generated_data]
-            if generated_data is None
-            else generated_data + [new_generated_data]
-        )
+        generated_data.append(new_generated_data)
```

Comment:
So, `new_generated_data` here is a list of Datasets which I think means we should be doing `generated_data.extend(new_generated_data)` instead of `.append`?

Reply:
Actually, looks like it was me who was reading this wrong, I think? marking this comment as resolved and will take another look as I'm attempting to track down some other bugs in this area.